src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs

```diff
@@ -212,6 +212,11 @@ public static Mullion Wrap(Autodesk.Revit.DB.Mullion ele, bool isRevitOwned)
            return Mullion.FromExisting(ele, isRevitOwned);
         }
 
+        public static DetailCurve Wrap(Autodesk.Revit.DB.DetailCurve ele, bool isRevitOwned)
```

Comment:
@Randy-Ma does the class show up in Dynamo? It has the [SupressImportIntoVM] attribute set.